<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>contrapartidas</class>
 <widget class="QDialog" name="contrapartidas">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>404</width>
    <height>314</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Contrapartidas de apuntes</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Cuenta contrapartida 1:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buscapushButton">
       <property name="maximumSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="keme.qrc">
         <normaloff>:/iconos/graf/viewer.xpm</normaloff>:/iconos/graf/viewer.xpm</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="auxiliarlineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="auxiliarlabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>- -</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Cuenta contrapartida 2:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buscapushButton_2">
       <property name="maximumSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="keme.qrc">
         <normaloff>:/iconos/graf/viewer.xpm</normaloff>:/iconos/graf/viewer.xpm</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="auxiliarlineEdit_2"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="auxiliarlabel_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>- -</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Valor segunda contrapartida:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="valorlineEdit"/>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox">
       <item>
        <property name="text">
         <string>Valor fijo</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Coeficiente</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="concepto_cuaderno_checkBox">
     <property name="text">
      <string>Tomar concepto de cuaderno importado</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Concepto:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="conceptolineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QCheckBox" name="verificar_checkBox">
       <property name="text">
        <string>Verificar primer asiento</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="guarda_valores_checkBox">
       <property name="text">
        <string>Guarda Valores</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancelar_pushButton">
       <property name="text">
        <string>&amp;Cancelar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="aceptar_pushButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>&amp;Aceptar</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>buscapushButton</tabstop>
  <tabstop>auxiliarlineEdit</tabstop>
  <tabstop>buscapushButton_2</tabstop>
  <tabstop>auxiliarlineEdit_2</tabstop>
  <tabstop>valorlineEdit</tabstop>
  <tabstop>comboBox</tabstop>
  <tabstop>concepto_cuaderno_checkBox</tabstop>
  <tabstop>conceptolineEdit</tabstop>
  <tabstop>verificar_checkBox</tabstop>
  <tabstop>guarda_valores_checkBox</tabstop>
  <tabstop>cancelar_pushButton</tabstop>
  <tabstop>aceptar_pushButton</tabstop>
 </tabstops>
 <resources>
  <include location="keme.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>cancelar_pushButton</sender>
   <signal>clicked()</signal>
   <receiver>contrapartidas</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>307</x>
     <y>252</y>
    </hint>
    <hint type="destinationlabel">
     <x>315</x>
     <y>262</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
